.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -360,23 +398,32 @@ main() {
     NAME_SPACE="showcase-ci-nightly"
     NAME_SPACE_RBAC="showcase-rbac-nightly"
     NAME_SPACE_POSTGRES_DB="postgress-external-db-nightly"
+    NAME_SPACE_AKS="showcase-aks-ci-nightly"
```

Comment:
new namespace is not needed, I guess idea is to deploy  existing namespace `showcase-ci-nightly` and `showcase-rbac-nightly` on AKS cluster.

Reply:
Right now, project that yarn runs the playwright tests for depends on the namespace name. So if we want different project for AKS, we need a different namespace, or change the script to use different value for that
https://github.com/janus-idp/backstage-showcase/blob/5b48b6fadb24b0297015e9c578add6341337f011/e2e-tests/package.json#L17
https://github.com/janus-idp/backstage-showcase/blob/5b48b6fadb24b0297015e9c578add6341337f011/.ibm/pipelines/openshift-ci-tests.sh#L258